boards/arm/nucleo_l4r5zi/nucleo_l4r5zi.dts

```diff
@@ -70,7 +70,7 @@ arduino_serial: &usart3 {};
 	status = "ok";
 };
 
-&usart3 {
+&lpuart1 {
 	current-speed = <115200>;
```

Comment:
You should add another node, note replace usart3. usart3 is used I think for the Arduino shield.

Reply:
Right, this is actually a hack that went in. Fixed